client/src/components/remixPaste.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const RemixPaste = () => {
+  const [data, setData] = useState(null);
+  const [title, setTitle] = useState("");
+  const [message, setMessage] = useState("");
+  const [isEditing, setIsEditing] = useState(false);
+  const [error, setError] = useState(null);
+
+  let { name } = useParams();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        if (name) {
+          const res = await fetch(`http://localhost:8000/find/${name}`);
+
+          if (!res) {
+            throw new Error("Fetch failed");
+          }
+          const file = await res.json();
+
+          setData(file);
+          setTitle(file.file[1]);
+          setMessage(file.show_file);
+        }
+      } catch (error) {
+        console.error(error);
+      }
```

Comment:
instead dont throw an error, but keep the catch, and in there set data and title to null

Reply:
```suggestion

          if (!res) {
            setData(null)
            setTitle(null)
                                }
          const file = await res.json();

          setData(file);
          setTitle(file.file[1]);
          setMessage(file.show_file);
        }
      } catch (error) {
        console.error(error);
      }
```

is this what you meant?